Rakefile

```diff
@@ -121,31 +112,25 @@ namespace :test do
       'test/functional/**/*_test.rb',
       'lib/plugins/inspec-*/test/functional/**/*_test.rb',
     ])
-    t.warning = false # This just complains about things in underlying libraries
+    t.warning = true
     t.verbose = true
```

Comment:
I screwed up with the ENV["V"] above. It should be everywhere. Can you fix that?